dev/heap_in_use.m

```diff
@@ -0,0 +1,9 @@
+heapuse=[];
+h=figure;
+ax=axes('parent',h);
+while true,
+  [s,out] = system("dcrps memstats dcrdata | grep heap-in-use | awk '{print $3}' | sed 's/(//'");
+  heapuse(end+1)=str2double(out);
+  plot(ax, heapuse); drawnow
+  pause(0.05)
+end
```

Comment:
GNU Octave script for live plotting `heap-in-use` (`runtime.MemStats.HeapInuse`). Uses [`dcrps`](https://github.com/dcrlabs/dcrps) and requires the binary to be called "dcrdata".